proptest-derive/tests/misc.rs

```diff
@@ -67,6 +67,21 @@ enum Quux {
     },
 }
 
+#[derive(Clone, Debug, Arbitrary)]
+struct MoreThanTenFields {
```

Comment:
Can you move this to a new file `structs.rs` and test some more edge cases (see the enums.rs test for the general structure for the tests)?

Reply:
I've done this and I've added the T13 case for the `Parameters` type. Are there any other edge cases you have in mind? (Empty structs are already checked for in `units.rs`)

Reply:
It's probably fine like this but if you want to be more thorough it might be good to test that you've exercised `if let [c] = head {` as well as two recursions down, e.g. `9 + 9 + 2` and also `9 + 9 + 1`.

Reply:
Good point! I've also added a test for a flat, non nested expansion.